tests/deploys.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/neotest"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/stretchr/testify/require"
+)
+
+type (
+	// RegContractNNSParams is a param struct for RegContractNNS func.
+	RegContractNNSParams struct {
+		// Name is a contract name. Name must not be a top level domain. For instance: netmap.
+		// The final registered domain name will be concatenated with `.neofs`.
+		Name         string
+		ContractHash util.Uint160
+		Email        string
+		Refresh      int64
+		Retry        int64
+		// Expire in milliseconds
+		Expire int64
+		TTL    int64
+	}
+)
+
+// NewRegContractNNSParams is a constructor for RegContractNNSParams.
+func NewRegContractNNSParams(name string, contractHash util.Uint160) RegContractNNSParams {
+	return RegContractNNSParams{
+		Name:         name,
+		ContractHash: contractHash,
+		Email:        "ops@nspcc.ru",
+		Refresh:      3600,
+		Retry:        600,
+		Expire:       int64(365 * 24 * time.Hour / time.Millisecond), // 1 year.
+		TTL:          3600,
+	}
+}
+
+// RegContractNNS make registration of the contract hash in nns contract.
+func RegContractNNS(t *testing.T, e *neotest.Executor, params RegContractNNSParams) {
+	nnsHash, err := e.Chain.GetContractScriptHash(1)
+	require.NoError(t, err)
+
+	nnsInv := e.CommitteeInvoker(nnsHash)
+
+	nnsInv.Invoke(t, true, "register", params.Name+".neofs", e.CommitteeHash, params.Email, params.Refresh, params.Retry, params.Email, params.TTL)
+	nnsInv.Invoke(t, nil, "addRecord", params.Name+".neofs", 16, params.ContractHash.StringLE())
+}
+
+// TickEpoch increase epoch value by one.
```

Comment:
```suggestion
// TickEpoch increases epoch value by one.
```
or just `increments`

Reply:
Updated